test/services/sync/strategies/TSP/TSPBaseSyncer.unit.test.js

```diff
@@ -0,0 +1,56 @@
+const { expect } = require('chai');
+const { BadRequest } = require('@feathersjs/errors');
+const mockery = require('mockery');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['mockery' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4569609)